### PR TITLE
[APPC-4515] Add Success screen

### PR DESCRIPTION
**Pull request by ricardo-frango**
**What does this PR do?**

   Added the success screen.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] MainFragment.kt
- [ ] SuccessScreen.kt
- [ ] iab_gift.json
- [ ] iab_success_view_background.json
- [ ] iab_transaction_success.json
- [ ] Animation.kt

**How should this be manually tested?**

  Start IAP flow to see the success screen.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4515](https://aptoide.atlassian.net/browse/APPC-4515)

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] Database migration (if applicable)?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Functional QA tests pass

[APPC-4515]: https://aptoide.atlassian.net/browse/APPC-4515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      iterations = LottieConstants.IterateForever,
+    )
+    Column(
+      modifier = Modifier.padding(start = 10.dp, end = 2.dp)
+    ) {
+      Text(
+        text = stringResource(id = R.string.bonus_received_body, bonus),
+        style = IAPTheme.typography.titleMedium,
+        color = IAPTheme.colors.onPrimary
+      ) // not fully correct according to figma
```
Comment (gustavoaptoide) [inline app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt:112]: Why not completely correct? Luis Approves this Color? IF not, we will change it to the correct color

File: app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      Text(
+        text = stringResource(R.string.gamification_purchase_body),
+        style = IAPTheme.typography.bodySmall,
+        color = IAPTheme.colors.smallText
+      ) // not fully correct according to figma
```
Comment (gustavoaptoide) [inline app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt:117]: Why not completely correct? Luis Approves this Color? IF not, we will change it to the correct color

**Review by gustavoaptoide — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt (hunk 1 above)
Comment (ricardo-frango) [reply app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt:112]: its the resource string that it's incorrect according to figma. that copy does not exist yet... the colors are ok

File: app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt (hunk 2 above)
Comment (ricardo-frango) [reply app/src/main/java/com/asfoundation/wallet/iab/presentation/SuccessScreen.kt:117]: its the resource string that it's incorrect according to figma. that copy does not exist yet... the colors are ok

**Review by gustavoaptoide — APPROVED** [review]

**Pull request merged**